src/core/cuda_utils.h

```diff
@@ -25,13 +25,16 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #pragma once
 
+#include <future>
 #include <set>
 #include "src/core/status.h"
 
 #ifdef TRITON_ENABLE_GPU
 #include <cuda_runtime_api.h>
 #endif  // TRITON_ENABLE_GPU
 
+#include "src/core/sync_queue.h"
```

Comment:
Why is this include down here and not right after src/core/status.h?

Reply:
Fixed